core/es_isolate.rs

```diff
@@ -130,15 +58,13 @@ pub struct ModuleInfo {
 /// as arguments. An async Op corresponds exactly to a Promise in JavaScript.
```

Comment:
The docs here don't make the distinction between EsIsolate and Isolate. It seems like the description ought to mention something about modules.